docs/source/en/tasks/image_classification.mdx

```diff
@@ -228,6 +276,144 @@ Once training is completed, share your model to the Hub with the [`~transformers
 </pt>
 </frameworkcontent>
 
+<frameworkcontent>
+<tf>
+
+<Tip>
+If you are unfamiliar with fine-tuning a model with Keras, check out the <a href="https://huggingface.co/docs/transformers/main/en/training#train-a-tensorflow-model-with-keras">basic tutorial</a> first!
+</Tip>
+
+To fine-tune a model in TensorFlow, follow these steps:
+1. define the training hyperparameters, and set up an optimizer and a learning rate schedule
+2. instantiate a pre-treined model
+3. convert a 🤗Dataset to a `tf.data.Dataset`
+4. compile your model
+5. add callbacks and use the `fit()` method to run the training
+6. upload your model to 🤗Hub to share with the community
+
+Start by defining the hyperparameters, optimizer and learning rate schedule:
+
+```py
+>>> from transformers import create_optimizer
+>>> import tensorflow as tf
+
+>>> batch_size = 16
+>>> num_epochs = 5
+>>> num_train_steps = len(food["train"]) * num_epochs
+>>> learning_rate = 3e-5
+>>> weight_decay_rate = 0.01
+
+>>> optimizer, lr_schedule = create_optimizer(
+...     init_lr=learning_rate,
+...     num_train_steps=num_train_steps,
+...     weight_decay_rate=weight_decay_rate,
+...     num_warmup_steps=0,
+... )
+```
+
+Then, load ViT with `TFAutoModelForImageClassification` along with the label mappings:
+
+```py
+>>> from transformers import TFAutoModelForImageClassification
+
+>>> model = TFAutoModelForImageClassification.from_pretrained(
+...     checkpoint,
+...     id2label=id2label,
+...     label2id=label2id,
+... )
+```
+
+Convert your datasets to the `tf.data.Dataset` format using the [`~datasets.Dataset.to_tf_dataset()`] and your `data_collator`:
+
+```py
+>>> # converting our train dataset to tf.data.Dataset
+>>> tf_train_dataset = food["train"].to_tf_dataset(
+...     columns=["pixel_values"], label_cols=["label"], shuffle=True, batch_size=batch_size, collate_fn=data_collator
+... )
+
+>>> # converting our test dataset to tf.data.Dataset
+>>> tf_eval_dataset = food["test"].to_tf_dataset(
+...     columns=["pixel_values"], label_cols=["label"], shuffle=True, batch_size=batch_size, collate_fn=data_collator
+... )
+```
+
+Configure the model for training with `compile()`:
```

Comment:
nit: possible suggestion, feel free to ignore. To keep spacing consistent (one new line before and after code block)

```suggestion
Configure the model for training with `compile()`:

```

Reply:
Fixed in 39a13a90375e52a9e373bf71d01309bc80531578